addons/mail/static/src/models/composer_view.js

```diff
@@ -299,6 +299,15 @@ registerModel({
                 markEventHandled(ev, 'Composer.closeEmojisPopover');
             }
         },
+        /**
+         * @param {CustomEvent} ev
```

Comment:
I'm not sure if `paste` is a custom event.

Reply:
Apparently this is `ClipboardEvent`. I just changed that.